MIGRATION.md

```diff
@@ -0,0 +1,7 @@
+# Migration Guides
+
+## Migrating from v1 to v2
+
+- Requires node 12 or later.
+- Removes the ability to compare bundle sizes of any ref pushed to Github. It is now only possible to compare the bundle sizes of releases. This is because the [Origami Build Service](https://www.ft.com/__origami/service/build/) drops support for bundling commit references for components which follow v2 of the Origami Component Specification.
```

Comment:
Good feature to remove, it does remove the ability to comment on pull-requests with the changes in js and css size but we can solve that another way.

One way to solve it is by publishing to npm every time a commit is push to a pull-request. The version in package.json could be the commit hash and we'd need to set the dist-tag to something other than 'latest' like we do for prereleases.

Reply:
👍 or I also liked @chee's idea, have obt figure out bundle sizes and commit results to the component